CMakeLists.txt

```diff
@@ -720,6 +725,24 @@ if(WITH_OPTIM)
                 set(WITH_POWER9 OFF)
             endif()
         endif()
+    elseif(BASEARCH_RISCV_FOUND)
+        if(WITH_RVV)
```

Comment:
There should be `check_rvv_intrinics` function that checks if the compiler supports `-march=rv64gcv`. And then that flag can be removed from the toolchain file.

Reply:
Thanks, it's done. We will use `set_property` to add compiler flags "-march=rv64gcv" to the files for rvv optimized kernel in the related pr.